exporter/exporterprofiles/go.mod

```diff
@@ -0,0 +1,68 @@
+module go.opentelemetry.io/collector/exporter/exporterprofiles
+
+go 1.21.0
+
+require (
+	github.com/stretchr/testify v1.9.0
+	go.opentelemetry.io/collector/component v0.105.0
+	go.opentelemetry.io/collector/consumer/consumertest v0.105.0
+	go.opentelemetry.io/collector/exporter v0.0.0-00010101000000-000000000000
```

Comment:
Can we use `v0.105.0` here?

Reply:
No, because a07e6c3e1038ee3685676c3cd9b63ac68d252d53 has not been released yet. So the required `internal` package isn't available in 0.105.0.

Reply:
Ah, I was removing the replace to have `go tidy` get the version. With the replace, it works.